src/transformers/tokenization_utils_base.py

```diff
@@ -1954,6 +1954,7 @@ def from_pretrained(
         from_pipeline = kwargs.pop("_from_pipeline", None)
         from_auto_class = kwargs.pop("_from_auto", False)
         commit_hash = kwargs.pop("_commit_hash", None)
+        from_gguf = kwargs.get("from_gguf", False)
```

Comment:
is this a bool or can it store a path to a file? (seems like a path)

Reply:
I agree it's not conssitent, it should be always a path, will change that

Reply:
Done !